2019-04-30 16:44:24,719 [DEBUG] Using GPU 0
2019-04-30 16:44:24,772 [DEBUG] CACHEDIR=/home/thinh/.cache/matplotlib
2019-04-30 16:44:24,775 [DEBUG] Using fontManager instance from /home/thinh/.cache/matplotlib/fontlist-v300.json
2019-04-30 16:44:24,899 [DEBUG] Loaded backend agg version unknown.
2019-04-30 16:44:32,834 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_03.h5
2019-04-30 16:44:32,835 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:44:41,393 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_04.h5
2019-04-30 16:44:41,393 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:44:48,745 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_14.h5
2019-04-30 16:44:48,745 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:44:56,119 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_18.h5
2019-04-30 16:44:56,119 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:03,465 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_19.h5
2019-04-30 16:45:03,465 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:10,880 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_21.h5
2019-04-30 16:45:10,880 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:18,263 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_22.h5
2019-04-30 16:45:18,263 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:25,701 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_23.h5
2019-04-30 16:45:25,702 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:33,098 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_24.h5
2019-04-30 16:45:33,098 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:45:40,514 [INFO] ==> VIDEO_ROOT_PATH/UCSD_ped1/testing_h5_t0/UCSD_ped1_32.h5
2019-04-30 16:45:40,514 [DEBUG] Predicting using logs/UCSD_ped1/jobs/f5846318-6990-4622-b584-ececf33a54d7/model_snapshot_e298_0.003159.h5
2019-04-30 16:46:59,218 [CRITICAL] Uncaught exception
Traceback (most recent call last):
  File "/home/thinh/Downloads/pycharm-community-2018.3.5/helpers/pydev/pydevd.py", line 1741, in <module>
    main()
  File "/home/thinh/Downloads/pycharm-community-2018.3.5/helpers/pydev/pydevd.py", line 1735, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/home/thinh/Downloads/pycharm-community-2018.3.5/helpers/pydev/pydevd.py", line 1135, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/thinh/Downloads/pycharm-community-2018.3.5/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/thinh/github/anomaly-detection-and-localization/start_test.py", line 73, in <module>
    visualize_score=True, visualize_frame=True)
  File "/home/thinh/github/anomaly-detection-and-localization/classifier.py", line 643, in test
    calc_auc_frame_overall(logger, dataset, n_videos[dataset], save_path, frame_level_error(rte))
  File "/home/thinh/github/anomaly-detection-and-localization/classifier.py", line 307, in calc_auc_frame_overall
    gt_vid = get_gt_vid(dataset, vid, frame_prediction[vid])
IndexError: index 10 is out of bounds for axis 0 with size 10
